### ISSUE TITLE
Optimize fire

### ISSUE THREAD

**Issue by StealthDrifter** [studio-minus/ppg-bugs-and-requests#802]
**Feature description**
Optimize fire so it doesn't lag people's pc's

**Feature purpose**
Make the game less laggy with multiple objects on fire

**Additional context**
dropping 80 frames with only 12 humans on fire is insane
dropping 40 frames with 7 humans on fire
like what

**Comment by mestiez** [comment]
Yes. The reason for the performance drop is that the fire is still using Unity's old disease ridden particle system, as do all other particle effects in the game. Fire, at the very least, should be using the newer GPU VFX system which would make it run orders of magnitudes faster
